src/probe/router.ts

```diff
@@ -23,11 +23,23 @@ const locationKeyMap = [
 ];
 
 export class ProbeRouter {
+	static hasIndex(socket: Socket, index: string) {
+		return socket.data.probe.index.some(v => v.includes(index.replace('-', ' ').trim().toLowerCase()));
+	}
+
+	static hasTag(socket: Socket, tag: string) {
+		return socket.data.probe.tags.some(({type, value}) => type === 'system' && value.includes(tag.trim().toLowerCase()));
```

Comment:
Might be better to normalize the data when building the probe?

Reply:
Here `value` is a data from probe it is already normalized. We are only normalizing user input (`tag` variable).

Reply:
True and that seems to be normalized on input as well? At least lowercase, not sure about trimming but that can be moved there as well.

Reply:
Removed toLowerCase, trim is necessary here because values from magic comes as string separated by '+' and will not be trimmed by Joi.